bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/HttpUtil.java

```diff
@@ -152,63 +157,70 @@ public static String executeUrl(String httpMethod, String url, Properties httpHe
 
         HttpClient client = new HttpClient();
```

Comment:
Reading the [docs](http://download.eclipse.org/jetty/stable-9/apidocs/org/eclipse/jetty/client/HttpClient.html), it says "HttpClient acts as a central configuration point " and "HttpClient transparently pools connections to servers".
I know that creating a new instance is usually an expensive operation and we are doing it here for every single request. I am wondering, if we can reuse the http client for all our calls? Or is there a good reason to have a separate instance for each?